centreon/www/widgets/src/centreon-widget-resourcestable/src/spec/ResourcesTable.cypress.spec.tsx

```diff
@@ -502,6 +523,97 @@ describe('Open tickets', () => {
     cy.contains('Ticket ID');
     cy.contains('Ticket subject');
     cy.contains('Opened on');
+    cy.contains('Action').should('be.visible');
+
+    cy.makeSnapshot();
+  });
+
+  it('displays a confirmation modal when a close ticket button is clicked for a service', () => {
+    render({
+      data: { resources },
+      options: {
+        ...resourcesOptions,
+        displayResources: 'withTicket',
+        isOpenTicketEnabled: true,
+        provider: { id: 1, name: 'Rule 1' },
+        selectedColumnIds: [
+          ...selectedColumnIds,
+          'ticket_id',
+          'ticket_subject',
+          'ticket_open_time',
+          'action'
+        ]
+      }
+    });
+
+    cy.waitForRequest('@getResources');
+
+    cy.contains('Action').should('be.visible');
+
+    cy.findAllByLabelText(labelCloseTicket).eq(0).click();
+
+    cy.contains(labelCloseATicket).should('be.visible');
+    cy.contains(labelTicketWillBeClosedInTheProvider).should('be.visible');
+
+    cy.contains(labelConfirm).click();
+
+    cy.waitForRequest('@postTicketClose').then(({ request }) => {
+      expect(request.body).equal(
+        JSON.stringify({
+          data: {
+            selection: '14;19',
+            rule_id: '1'
+          }
+        })
+      );
+    });
+
+    cy.contains(labelTicketClosed).should('be.visible');
+
+    cy.makeSnapshot();
+  });
```

Comment:
The test description only covers the first use case (displaying the confirmation dialog). Splitting this test function could lead to better coherence and ensure the snapshot tests captures the modal a well